### ISSUE TITLE
No skipFailureScreen option

### ISSUE THREAD

**Issue by mpp293** [sumup/sumup-android-sdk#53]
We could use such function, especially that the skipSuccessScreen is already available.
In case of failed transaction one can still read the error code programmatically and present the status on own screen.

**Comment by jahenry** [comment]
Hi there

For transaction status
SDK: Returned in transaction response
App Switch: Callback URL success or fail

For a 2nd source of truth, please see SumUp's Transaction History API: http://docs.sumup.com/rest-api/transactions-api/

Thanks

**Comment by vladaman** [comment]
I'd like add +1 to this. We'd like to see this skipFailureScreen as well. From Henry's response I don't see any solution to this. Can you please clarify?

Basically we'd like to skip the Failure screen and show error directly in our app. Thanks.

**Comment by jahenry** [comment]
@vladaman No you currently cannot skipfailurescreen.  skipsuccessscreen is the only option available at present

**Comment by PatrykGlowacki** [comment]
Yes @nstubbe, you have right. I waiting for skipFailureScreen too. FailureScreen make a lot of problems for end-users.

**Comment by openmove** [comment]
news about this? 
skipFailureScreen would be very useful.

**Comment by JullianSU** [comment]
Hi Everyone, 

Sorry for the delay in response but we have not moved forward on that topic. The success/failure screen is still part of our SDK and it is not in the immediate roadmap to remove them. 

We will keep you informed if it is something that we want to do. 

Many thanks,

**Comment by miggat** [comment]
2 years later and still nothing? This is making the app really hard to use.

**Comment by PatrykGlowacki** [comment]
I gave up the project with this device. Still nothing

czw., 19 sie 2021 o 18:35 miggat ***@***.***> napisał(a):
